templates/includes/sidebar.html

```diff
@@ -1,4 +1,9 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <p><strong>User:</strong> <a href="{% url 'taxi:driver_detail' pk=user.id %}">{{ user.username }}</a></p>
```

Comment:
If I didn't miss something, there is no "Login" link to login page in case user is not authenticated, but you have it on screenshots🤨